psycop/common/model_training_v2/trainer/preprocessing/steps/filters.py

```diff
@@ -16,3 +19,58 @@ def __init__(self, min_age: int, max_age: int, age_col_name: str):
 
     def apply(self, input_df: PolarsFrame_T0) -> PolarsFrame_T0:
         return input_df.filter((self.age >= self.min_age) & (self.age <= self.max_age))
+
+
+@BaselineRegistry.preprocessing.register("lookbehind_combination_filter")
+class LookbehindCombinationFilter(PresplitStep):
```

Comment:
```suggestion
class LookbehindFilter(PresplitStep):
```

Reply:
I guess we might need another col filter based on lookbehind, e.g. `LookbehindExceedsLookbehindFilter` so maybe we should stick to the more specific name.

Reply:
Hmm, I see. Perhaps add something with column in the name (both for the registry and for the class). Will make the configs much easier to read in the future :-)

Other than that, ready for merge!